doc/source/examples/multiphysics/tracer-in-static-mixer/tracer-in-static-mixer.rst

```diff
@@ -0,0 +1,295 @@
+
+--------------
+Parameter File
+--------------
+
+Simulation 1: Flow resolution
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Simulation control
+******************
+
+.. code-block:: text
+
+    subsection simulation control
+      set method    = bdf1
+      set time end  = 40e-4
+      set time step = 1e-4
+
+      set adapt = false
+
+      set output path      = ./output/
+      set output name      = output
+      set output control   = iteration
+      set output frequency = 5
+    end
+
+#. We use a ``bdf1`` time integration scheme with a short ``time step = 1e-4``. After ``time end = 40e-4``, we consider that the velocity field has reached a steady-state.
+#. We do not allow time step adaptation, since it is a parameter that we control ourselves between simulation: ``adapt = false``.
+
+Restart
+******************
+
+We use the checkpoint/restart mechanism of Lethe to achieve these simulations.
+
+.. code-block:: text
+
+    subsection restart
+      set checkpoint = true
+      set frequency  = 5
+      set filename   = restart
+      set restart    = false
+    end
+
+#. We ``checkpoint`` the simulation at each ``5`` time steps, both for safety (each time step takes a long time to complete) and for reuse in the second simulation.
+#. We do not ``restart`` the simulation, unless it is stopped before it reaches ``time end``.
+
+Multiphysics
+******************
+
+Both ``fluid dynamics`` and ``tracer`` are enabed for the first simulation, although nothing happens tracer-wise.
+
+.. code-block:: text
+
+    subsection multiphysics
+      set fluid dynamics = true
+      set tracer         = true
+    end
+
+Physical Properties
+*******************
+
+The physical properties here are defined using centimeter as the length-unit and seconds as the time-unit. We consider that we have a passive tracer in water.
+
+.. code-block:: text
+
+    subsection physical properties
+      subsection fluid 0
+        set kinematic viscosity      = 0.01
+        set tracer diffusivity model = immersed solid tanh
+        subsection immersed solid tanh
+          set tracer diffusivity inside  = 1e-10
+          set tracer diffusivity outside = 1e-5
+          set thickness                  = 5e-1
+        end
+      end
+    end
+
+#. The ``tracer diffusivity model`` is ``immersed solid tanh``. This model is used for tracer simulations involving immersed solids when using ``lethe-fluid-sharp``.
+#. The ``tracer diffusivity outside`` is ``1e-5``, as this is a typical value for a passive tracer in a liquid.
+#. The ``tracer diffusivity inside`` is set to ``1e-10``. We desire no diffusivity inside the solid, and this value is low enough to achieve this effect while providing numerical stability.
+#. The ``thickness`` is ``5e-1``. At the scale of the problem, this provides a smooth transition without generating oscillations between liquid and solid phases.
```

Comment:
```suggestion
#. The ``tracer diffusivity model`` is ``immersed solid tanh``. This model is used in ``lethe-fluid-sharp`` for tracer flow percolating immersed solids.
#. The ``tracer diffusivity outside`` is ``1e-5``, as this is a typical value for a passive tracer in a liquid.
#. The ``tracer diffusivity inside`` is set to ``1e-10``. The low value prevents diffusivity inside the solid while providing numerical stability.
#. The ``thickness`` is ``5e-1``. At the scale of the problem, this provides a smooth transition without generating oscillations between liquid and solid phases.
```

Reply:
Missing units.

Reply:
Is this based on previous tests? It would be nice to mention why these values.

Reply:
It wasn't based on previous tests, more on value we thought were appropriate.